src/plugin/plugin.cc

```diff
@@ -238,10 +238,20 @@ void plugin::Zeek_Spicy::Plugin::registerPacketAnalyzer(const std::string& name,
 void plugin::Zeek_Spicy::Plugin::registerType(const std::string& id, const ::zeek::TypePtr& type) {
     auto [ns, local] = parseID(id);
 
-    if ( ::zeek::detail::lookup_ID(local.c_str(), ns.c_str()) ) {
-        // Note, this is unlikely  to trigger, because we run before Zeek
-        // registers its own IDs (but then Zeek will catch it).
-        reporter::error(hilti::rt::fmt("attempt to overwrite already defined Zeek type '%s'", id));
+    if ( auto old = ::zeek::detail::lookup_ID(local.c_str(), ns.c_str()) ) {
```

Comment:
Style nit: No need to make a copy of the `IDPtr` here.
```suggestion
    if ( const auto& old = ::zeek::detail::lookup_ID(local.c_str(), ns.c_str()) ) {
```